.github/actions/integration-tests/action.yml

```diff
@@ -0,0 +1,15 @@
+name: Integration Tests
+description: Run Integration Tests
+runs:
+  using: "composite"
+  steps: 
+    - run: sudo sysctl -w vm.max_map_count=262144
```

Comment:
I don't think this is needed anymore with ES 7 (it was with ES 5 and I think we never updates this part of the CI since)

Reply:
It is still needed: https://www.elastic.co/guide/en/elasticsearch/reference/current/_maximum_map_count_check.html

Reply:
I don't need to on my computer, did you try on yours ? 
![image](https://user-images.githubusercontent.com/4447392/101911994-73a74100-3bc1-11eb-9375-3fdda98e79ea.png)